ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/apps/AppsMessages.properties

```diff
@@ -54,3 +54,6 @@ url = URL:
 appUrl = App URL
 copyAppUrl = Copy App URL
 copyAppSuccessMessage = ''{0}'' was successfully copied, and the copy is now open for editing.
+share = Share
+shareCollab = Share with Collaborators...
+sharePublic = Share with Public...
```

Comment:
This seems a bit stilted -- maybe "Make Public" or so? I think it doesn't matter much to the users that it's being made public by way of a sharing system, unlike with the collaborators where they'd probably go looking for something that says "share" on it.